module.exports = (fn) => {
  fn().catch((error) => {
    console.log('Something went very wrong ğŸ”¥ğŸ§¯', error);
    process.exit(1)
  });
};
